l10n/configs/pontoon.toml

```diff
@@ -339,6 +339,24 @@ locales = [
 [[paths]]
     reference = "en/privacy/*.ftl"
     l10n = "{locale}/privacy/*.ftl"
+[[paths]]
+    reference = "en/privacy/firefox.ftl"
+    l10n = "{locale}/privacy/firefox.ftl"
+    locales = [
+        "cs",
+        "de",
```

Comment:
You can drop de, fr, id, ja because they're managed through Smartling
https://github.com/mozilla-l10n/www-l10n/blob/master/configs/smartling-config.json